packages/plugin-ext/src/main/browser/documents-main.ts

```diff
@@ -189,6 +238,15 @@ export class DocumentsMainImpl implements DocumentsMain {
         return false;
     }
 
+    async $tryOpenDocument(uri: UriComponents): Promise<boolean> {
+        const ref = await this.modelService.createModelReference(new URI(CodeURI.revive(uri)));
+        if (ref.object) {
+            this.modelReferenceCache.add(ref);
+            return true;
+        }
+        return false;
```

Comment:
i don't think it can happen, but then we should call  `ref.dispose()` here as well